src/parameter_declarations.ts

```diff
@@ -242,17 +241,6 @@ Parameter.declare({
 Parameter.declare({
 	name: LocalPathParameter,
 	aliases: ['--local-path'],
-	description: `the path on the local filesystem for the file that will be uploaded
```

Comment:
Isn't this going to destroy the documentation for `upload-file`'s --local-path use case?

Reply:
This should fix that. 744a5e097b4d247abd5d218e2a8210ecb048b7b3